extension/src/test/suite/repository/sourceControlManagement.test.ts

```diff
@@ -262,12 +266,14 @@ suite('Source Control Management Test Suite', () => {
 
       expect(mockShowWarningMessage).to.be.calledOnce
       expect(mockCheckout).not.to.be.called
-      expect(mockGitReset).not.to.be.called
+      expect(mockExecuteProcess).not.to.be.called
     })
 
     it('should reset the workspace if the user confirms they want to', async () => {
       const mockCheckout = stub(CliExecutor.prototype, 'checkout').resolves('')
-      const mockGitReset = stub(ProcessExecution, 'executeProcess').resolves('')
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const mockExecuteProcess = stub(Cli.prototype as any, 'executeProcess')
```

Comment:
Move to a util so that we only have one `elint-disable`?

Reply:
I'll sort this out for good by adding a test util as a follow-up today.